images/kubernetesonarm/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM larmog/armhf-alpine-java:jdk-8u73
+
+EXPOSE 9200 9300
```

Comment:
No `EXPOSE` adds an extra layer that's only required for `docker run -P`. The same for volume. I'm trying to make the layers as few as possible.